CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+
+- Specific. Include as much details as possible: which version, what environment, what configuration, etc.
+
+- Reproducible. Include the steps to reproduce the problem. We understand some issues might be hard to reproduce, please include the steps that might lead to the problem.
+
+- Isolated. Please try to isolate and reproduce the bug with minimum dependencies. It would significantly slow down the speed to fix a bug if too many dependencies are involved in a bug report. Debugging external systems that rely on oria-operator is out of scope, but we are happy to provide guidance in the right direction or help with using oria-operator itself.
+
+- Unique. Do not duplicate existing bug report.
+
+- Scoped. One bug per report. Do not follow up with another bug inside one report.
+
+It may be worthwhile to read [Elika Etemad’s article on filing good bug reports](https://fantasai.inkedblade.net/style/talks/filing-good-bugs/) before creating a bug report.
+
+We might ask for further information to locate a bug. A duplicated bug report will be closed.
+
+## Contribution flow
+
+This is a rough outline of what a contributor's workflow looks like:
+
+- Create a new branch based from the `main` branch.
+- Make commits of logical units.
+- Make sure commit messages are in the proper format (see below).
+- Push branch with changes to a personal fork of the repository.
+- Submit a pull request to operator-framework/oria-operator.
+- The PR must be approved by an approver and receive a LGTM from a reviewer. Approvers and reviewers can be found in the OWNERS file.
+
+Thanks for contributing!
+
+### Code style
+
+The coding style suggested by the Go community is used in oria-operator. See the [style doc][golang-style-doc] for details.
+
+Please follow this style to make oria-operator easy to review, maintain and develop.
+
+### Format of the commit message
+
+We follow a rough convention for commit messages that is designed to answer two
+questions: what changed and why. The subject line should feature the what and
+the body of the commit should describe the why.
+
+```
+scripts: add the test-cluster command
+
+this uses tmux to setup a test cluster that can easily be killed and started for debugging.
+
+Fixes #38
+```
+
+The format can be described more formally as follows:
+
+```
+<subsystem>: <what changed>
+<BLANK LINE>
+<why this change was made>
+<BLANK LINE>
+<footer>
+```
+
+The first line is the subject and should be no longer than 70 characters, the second line is always blank, and other lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various git tools.
+
+## Documentation
+
+Most contributions involve some sort of documentation. Currently all documentation resides in the `docs/` folder or a succinct section in the README. When making contributions, it is important to ensure that all documentation is updated as necessary.
```

Comment:
Most contributions involve some sort of documentation. Currently, all documentation resides in the `docs/` folder or a succinct section in the README. When making contributions, it is important to ensure that all documentation is updated as necessary.

Reply:
I *think* this is resolved with commit https://github.com/operator-framework/oria-operator/pull/73/commits/c387d7632b2f440788ea0a5cb39293fb9ddbbea2